app/Console.php

```diff
@@ -94,6 +94,18 @@ class Console extends Model
             'provider' => 'Digital Ocean',
             'route' => 'digital-ocean',
         ],
+        [
+            'name' => 'Apple Developer Program',
+            'url' => 'https://developer.apple.com/account',
+            'provider' => 'Apple',
+            'route' => 'apple/developer-program',
```

Comment:
what would you think if this was instead "developer" and the other was "connect" just for speed?

Reply:
Are you asking about the routes being those shorter versions? I think that would be fine, could also see dropping "Apple" from the first label to just be "Developer Program" to be more concise as well. 

I'll update the PR.

Reply:
@jonathanburton I was just thinking the routes; I don't mind it on the label, since that's its official title; you can see we kept it that way with Google:

<img width="777" alt="image" src="https://user-images.githubusercontent.com/151829/87840811-8c5c3680-c86f-11ea-8e04-ee61bd48bc26.png">

Thanks!

Reply:
@mattstauffer I've updated the PR with the more concise routes.